src/App.tsx

```diff
@@ -1,61 +1,125 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useEffect, useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState([...todosFromServer]);
+  const [task, setTask] = useState('');
+  const [user, setUser] = useState('Choose a user');
+  const [newTitle, setNewTitle] = useState(true);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [selectedUser, setSelectedUser] = useState(true);
+  const [hasUserError, setHasUserError] = useState(false);
+
+  const handleAdd = () => {
+    setHasTitleError(!newTitle);
+    setHasUserError(!selectedUser);
+
+    const currentTodos = [...todos];
+
+    if (newTitle && selectedUser) {
+      const largest = currentTodos
+        .sort((a, b) => a.id - b.id)[currentTodos.length - 1];
+
+      const searchedUser = usersFromServer.find(el => el.name === user);
+
+      const userId = searchedUser ? searchedUser.id : 0;
```

Comment:
Will be better to show some error, if there is no selected user on the server instead of creating todo with user id = 0